jjb/jobs/qcs-nightly-automation.yaml

```diff
@@ -0,0 +1,43 @@
+- job:
+    name: 'qcs-nightly-automation'
+    node: 'f25-np'
+    triggers:
+        - timed: '@midnight'
+    scm:
+        - git:
+            url: https://github.com/quipucords/quipucords.git
+            basedir: quipucords
+            branches:
+                - '*/master'
+            skip-tag: true
+    wrappers:
+        - ssh-agent-credentials:
+            users:
+                - '390bdc1f-73c6-457e-81de-9e794478e0e'
+    builders:
+        - shining-panda:
+            build-environment: virtualenv
+            python-version: System-CPython-3
+            nature: shell
+            command: |
+                cd quipucords
+                sudo dnf install -y python-tools
+                pip install -r requirements.txt
+
+                pip install pytest-django pytest-pythonpath
+
+                cat > quipucords/pytest.ini <<EOF
+                [pytest]
+                DJANGO_SETTINGS_MODULE = quipucords.settings
+                python_files = tests.py test_*.py *_tests.py *_test.py
+                python_path = quipucords
+                EOF
+
+                cd quipucords
+                set +e
+                pytest -v --junit-xml ${WORKSPACE}/junit.xml
```

Comment:
I think that we should be running camayoc tests instead of the quipocord's unit tests. So It will be better to leave this PR open until we have the initial tests on camayoc and update this PR to run them instead.

Reply:
The reason why this is because the unittests is already run on travis for every PR and branch on quipucords repository.